Sources/DistributedActorsTestKit/TestProbes.swift

```diff
@@ -58,6 +61,8 @@ public final class ActorTestProbe<Message: Codable>: @unchecked Sendable {
         self.settings.expectationTimeout
     }
 
+    private let system: ClusterSystem
```

Comment:
do we need to store the system, or just initialize the internal test probe with it is enough? Would be good to not store the system if we can hm

Reply:
not storing anymore. just use it in `init`.